RELEASE-NOTES.md

```diff
@@ -1,5 +1,207 @@
+    my_background_task.delay("ford")
+```
+
+See the following pull requests for implementation details:
+- https://github.com/PrefectHQ/prefect/pull/13695
+- https://github.com/PrefectHQ/prefect/pull/13692
+- https://github.com/PrefectHQ/prefect/pull/13681
+- https://github.com/PrefectHQ/prefect/pull/13662
+- https://github.com/PrefectHQ/prefect/pull/13653
+- https://github.com/PrefectHQ/prefect/pull/13643
+- https://github.com/PrefectHQ/prefect/pull/13589
+- https://github.com/PrefectHQ/prefect/pull/13684
+- https://github.com/PrefectHQ/prefect/pull/13676
+- https://github.com/PrefectHQ/prefect/pull/13276
+- https://github.com/PrefectHQ/prefect/pull/13611
+- https://github.com/PrefectHQ/prefect/pull/13547
+
+### Transactional semantics - Rollback and commit hooks that facilitate idempotent python code
+
+```python
+from prefect import flow, task
+from prefect.transactions import transaction
+@task
+def first_task():
+    print('first')
+
+@first_task.on_rollback
+def roll(txn):
+    print('rolling back')
+
+@task
+def second_task():
+    raise RuntimeError("oopsie")
+
+@flow
+def txn_flow():
+    with transaction():
+        first_task()
+        second_task()
+if __name__ == "__main__":
+    txn_flow()
+
+See the following pull requests for implementation details:
+- https://github.com/PrefectHQ/prefect/pull/13559
+- https://github.com/PrefectHQ/prefect/pull/13534
+- https://github.com/PrefectHQ/prefect/pull/13535
+- https://github.com/PrefectHQ/prefect/pull/13480
+- https://github.com/PrefectHQ/prefect/pull/13452
+- https://github.com/PrefectHQ/prefect/pull/13450
+- https://github.com/PrefectHQ/prefect/pull/13484
+- https://github.com/PrefectHQ/prefect/pull/13454
+- https://github.com/PrefectHQ/prefect/pull/13477
+- https://github.com/PrefectHQ/prefect/pull/13431
+- https://github.com/PrefectHQ/prefect/pull/13264
+- https://github.com/PrefectHQ/prefect/pull/13337
+- https://github.com/PrefectHQ/prefect/pull/13456
+- https://github.com/PrefectHQ/prefect/pull/13572
+- https://github.com/PrefectHQ/prefect/pull/13582
+- https://github.com/PrefectHQ/prefect/pull/13627
+- https://github.com/PrefectHQ/prefect/pull/13568
+- https://github.com/PrefectHQ/prefect/pull/13438
+- https://github.com/PrefectHQ/prefect/pull/13573
+- https://github.com/PrefectHQ/prefect/pull/13578
+- https://github.com/PrefectHQ/prefect/pull/13414
+
+### Open source Events and Automations - Trigger actions, such as sending notifications, pausing schedules, starting flow runs and more in response to Prefect events.
+
+See the following pull requests for implementation details:
+- https://github.com/PrefectHQ/prefect/pull/13293
+- https://github.com/PrefectHQ/prefect/pull/13521
+- https://github.com/PrefectHQ/prefect/pull/13335
+
+### More flexible variables and new artifact types - Variables can now be any JSON compatible type including dicts, lists, and integers. Progress and Image artifacts make it easy to add visual annotations to your flow run graph.
```

Comment:
When rendered, it looks a bit odd to be including the highlight sentence in the heading to me